skiko/build.gradle.kts

```diff
@@ -1679,4 +1710,45 @@ tasks.withType<org.jetbrains.kotlin.gradle.tasks.KotlinNativeCompile>().configur
     // https://youtrack.jetbrains.com/issue/KT-56583
     compilerOptions.freeCompilerArgs.add("-XXLanguage:+ImplicitSignedToUnsignedIntegerConversion")
     kotlinOptions.freeCompilerArgs += "-Xopt-in=kotlinx.cinterop.ExperimentalForeignApi"
-}
\ No newline at end of file
+}
+
+abstract class AbstractImportGeneratorCompilerPluginSupportPlugin(
```

Comment:
it seems it might sense to keep this in buildSrc

Reply:
Moved to buildSrc.